routes.py

```diff
@@ -1,8 +1,14 @@
+"""
```

Comment:
All these new docstrings were added to pass lint. Also, because we should have been adding docstrings in the first place.

Reply:
Nice, I'm a big fan of docstrings. They belong at the top of a file before import statements, as well as above every function right? Do we put docstrings anywhere else?

Reply:
You're correct. Though, for functions and methods, you put them between the declaration and the body. You can also give classes docstrings.